### DISCUSSION TITLE
小爱音箱能否通过输入播报语音和输入文字交互

### DISCUSSION THREAD

**Discussion by caissonfiv** [XiaoMi/ha_xiaomi_home#370] [Ideas]
我记得以前的一些插件可以做到，输入文本后小爱音箱pro可以朗读出来，还有输入文本可以和小爱音箱pro交互。

**Comment by huhuime** [comment]
![image](https://github.com/user-attachments/assets/0737f011-a2e7-40b1-8b95-3ad9729b3708)
有的，就是教程没写明

**Reply by caissonfiv** [reply]
能不能给点思路

**Comment by zghnwsq** [comment]
可以的，是text实体，通过设定值，但是现在输入参数有点反人类，应该有相关pr了吧
参数格式是  ["文本"]

**Comment by rogueme** [comment]
播放文本格式为:["文本"],例:["我是小爱童鞋"]
执行文本指令格式为:["指令文本",true/false(是否静默)],例:["打开电脑",true],这这样执行后不会有反馈,["打开电脑",false],这样小爱童鞋会将结果反馈出来!

**Comment by hzcoolwind** [comment]
怎么用这个notify 播报，不会用啊， 调用哪个服务？

**Reply by caibinqing** [reply]
`notify.send_message`

**Comment by refined-fish** [comment]
就是一楼的方法，只是不像第三方插件直接给了输入框

**Comment by starxjys** [comment]
开发者工具->动作->notify.send_message->选择对应实体->["这是一条语音播报"]

**Reply by weiyifeng73** [reply]
麻烦问一下，要是想让小爱播放的内容的字符串是个变量，那应该如何设置？我打算用node-red里的action节点，但是感觉就只能输入预先设定的内容，不能输出可变的字符串

**Reply by starxjys** [reply]
node-red的action节点
Action   notify.send_message
entity   小爱音箱的播放文本实体
data   json格式   {"message" : "{{payload}}"}

注意上一个消息体的payload是文本内容，可以通过函数构建，或者用辅助元素文本框实体获得
